test/cypress/visual/Dashboards.cy.js

```diff
@@ -3,39 +3,59 @@ describe('Dashboards climate demo visual tests', () => {
         cy.intercept('/**/*.csv').as('getData');
         cy.visit('/dashboards/demo/climate');
         cy.wait('@getData', {timeout: 100000}) // wait for data to be laoded
-    })
+    });
 
     it('Climate demo', () => {
-        cy.boardRendered()
-        cy.get('#demo-content')
-            .compareSnapshot('dashboard-climate-loaded', 0.1);
-    })
+        cy.boardRendered();
+        cy.get('g.highcharts-markers.highcharts-series-1.highcharts-mappoint-series')
+            .children('.highcharts-point').should('have.length', 30);
+        cy.get('#demo-content').compareSnapshot('dashboard-climate-loaded', 0.1);
+    });
 
     it('edit mode', () => {
-        cy.toggleEditMode()
-        cy.get('.highcharts-dashboards-component').first().click()
+        cy.toggleEditMode();
+        cy.get('.highcharts-dashboards-component').first().click();
         cy.get('#demo-content')
             .compareSnapshot('dashboard-climate-edit-mode', 0.1);
+    });
+
+    it('small screen vertical', () => {
+        cy.toggleEditMode();
+        cy.viewport(375, 812).wait(500);
+        cy.get('#demo-content').compareSnapshot('dashboard-climate-mobile-vertical', 0.1);
+    });
 
-    })
+    it('small screen horizontal', () => {
+        cy.viewport(812, 375).wait(500);
+        cy.get('#demo-content').compareSnapshot('dashboard-climate-mobile-horizontal', 0.1);
+    });
 });
 
 describe('Test the rest', () => {
     const DEMOS_TO_VISUALLY_TEST = [
-        '/dashboards/demo/minimal'
-    ]
-
+        '/dashboards/demo/minimal',
+        '/dashboards/responsive/responsive-breakpoints'
+    ];
+    
     for (const demo of DEMOS_TO_VISUALLY_TEST) {
+        const name = demo.replace('/', '').replace(/\//g, '-');
+
         it('visually comparison after load ' + demo, () => {
             cy.visit(demo);
             cy.boardRendered();
 
-            cy.get('#demo-content')
-                .compareSnapshot(
-                    demo.replace('/', '')
-                        .replace(/\//g, '-') + '-loaded',
-                    0.1
-                );
-        })
+            cy.get('#demo-content').compareSnapshot(name + '-loaded', 0.1);
+        });
+
+        it('small screen vertical ' + demo, () => {
+            cy.viewport(375, 812).wait(500);
```

Comment:
Why do we need to wait here? Because of animations of layout or highcharts?

Reply:
Because of the layout, which needs some time to keep up with the reflow.
![image](https://github.com/highcharts/highcharts/assets/46498046/a241096c-504c-40dc-89ac-ed0770eb7067)

Reply:
Have you tried to disable all animations (also in CSS) ?